code/modules/atmospherics/components/unary/vent_pump.dm

```diff
@@ -62,8 +62,8 @@
 	icon_state = "map_vent_in"
 	external_pressure_bound = 0
 	external_pressure_bound_default = 0
-	internal_pressure_bound = MAX_PUMP_PRESSURE
-	internal_pressure_bound_default = MAX_PUMP_PRESSURE
+	internal_pressure_bound = 2000
```

Comment:
I don't think reverting back to 2k is a good idea. The whole point was to upgrade this to allow for better air transfer from tanks. I think dis value should remain 15k while the one below 2k. I think that should allow syphon vents to have max 15k output while remaining with 2k default start output. Also I would change max output on computer to this value, which I forgot to change myself

Reply:
2000 default is fine. i'll change the max to 15000.